__test__/api/comment-vote.test.js

```diff
@@ -28,17 +28,37 @@ describeIfEndpoint(
         });
       expect(res.status).toBe(200);
       expect(res.body.data.commentId).toEqual(String(comment._id));
-      expect(res.body.data.numOfVotes).toEqual(1);
-      expect(res.body.data.numOfUpVotes).toEqual(1);
-      expect(res.body.data.numOfDownVotes).toEqual(0);
+      expect(res.body.data.numOfVotes).toBeTruthy();
+      expect(res.body.data.numOfUpVotes).toBeTruthy();
+      expect(res.body.data.numOfDownVotes).toBeTruthy();
+    });
+
+    test("Should reject unknown vote type", async () => {
+      const comment = new CommentModel({
+        content: "this is a comment",
+        ownerId: "useremail3@email.com",
+        origin: "123123",
+        applicationId: global.application._id,
+      });
+      await comment.save();
+
+      const res = await request.patch(`/comments/${comment._id}/votes`).send({
```

Comment:
I thought this was now PATCH /comments/:commentId/votes/upvotes or PATCH /comments/:commentId/votes/downvotes. So there will be no invalid vote anymore. you are either upvoting or downvoting

Reply:
lol, let me remove that then